backend/package.json

```diff
@@ -22,6 +22,7 @@
   },
   "homepage": "https://github.com/hhe-wtag/bidify#readme",
   "dependencies": {
+    "bcryptjs": "^2.4.3",
```

Comment:
This version of bcrypt iss built with JavaScript and last updated version was made 8 years ago. I am suggesting to use https://www.npmjs.com/package/bcrypt which is built with C++ resulting in faster and more efficient.

Reply:
@hhe-wtag why are we using bcryptjs then instead of bcrypt? Was it specifically mentioned to use this in the requirements?

Reply:
@aht-wtag no bhaia there was no instruction on that. This PR is made by Mumu. I requested to change `bcryptjs` to `bcrypt` instead.

Reply:
I just overlooked the package as it was functional i worked with it. I am updating my code as bycrpt is recent. Thank you!